modules/kubernetes/grafana-agent/charts/grafana-agent-extras/templates/grafanaagent.yaml

```diff
@@ -22,3 +22,9 @@ spec:
     instanceSelector:
       matchLabels:
         agent: grafana-agent-logs
+  resources:
```

Comment:
I would make these into helm variabels. Not that we should expose them as variabels in terraform. But it will make it easier to visualise all the values in one central file.